### ISSUE TITLE
Error On Install

### ISSUE THREAD

**Issue by dev0x10** [tgriesser/knex#1735]
npm ERR! Linux 4.4.0-38-generic
npm ERR! argv "/usr/bin/nodejs" "/usr/bin/npm" "install"
npm ERR! node v6.7.0
npm ERR! npm  v3.10.3
npm ERR! code ELIFECYCLE

npm ERR! knex@0.12.4 postinstall: `node ./scripts/build.js lib "npm run build"`
npm ERR! Exit status 1
npm ERR!
npm ERR! Failed at the knex@0.12.4 postinstall script 'node ./scripts/build.js lib "npm run build"'.
npm ERR! Make sure you have the latest version of node.js and npm installed.
npm ERR! If you do, this is most likely a problem with the knex package,
npm ERR! not with npm itself.
npm ERR! Tell the author that this fails on your system:
npm ERR!     node ./scripts/build.js lib "npm run build"
npm ERR! You can get information on how to open an issue for this project with:
npm ERR!     npm bugs knex
npm ERR! Or if that isn't available, you can get their info via:
npm ERR!     npm owner ls knex
npm ERR! There is likely additional logging output above.

npm ERR! Please include the following file with any support request:
npm ERR!     /home/ooo/admin/npm-debug.log

I revert back to 0.12.1 and works fine
Btw I tested with 0.12.3 and no problem too.

**Comment by crvv** [comment]
`npm i knex@0.12.3` works

**Comment by Znarkus** [comment]
This is the error we're getting.

```
> knex@0.12.4 postinstall /node_modules/knex
> node ./scripts/build.js lib "npm run build"

module.js:457
    throw err;
    ^

Error: Cannot find module '/node_modules/knex/scripts/build.js'
    at Function.Module._resolveFilename (module.js:455:15)
    at Function.Module._load (module.js:403:25)
    at Module.runMain (module.js:590:10)
    at run (bootstrap_node.js:394:7)
    at startup (bootstrap_node.js:149:9)
    at bootstrap_node.js:509:3
```

**Comment by love8587** [comment]
I also found out it errors in my AWS elastic beanstalk env.

```
[2016-10-12T13:20:25.699Z] INFO  [14749] - [Application update app-c098-161012_211948@74/AppDeployStage0/AppDeployPreHook/50npm.sh] : Activity execution failed, because: + /opt/elasticbeanstalk/containerfiles/ebnode.py --action npm-install
  npm WARN optional dep failed, continuing fsevents@1.0.14
  npm WARN deprecated minimatch@2.0.10: Please update to minimatch 3.0.2 or higher to avoid a RegExp DoS issue

  > knex@0.12.4 postinstall /tmp/deployment/application/node_modules/knex
  > node ./scripts/build.js lib "npm run build"

  module.js:327
      throw err;
      ^

  Error: Cannot find module '/tmp/deployment/application/node_modules/knex/scripts/build.js'
      at Function.Module._resolveFilename (module.js:325:15)
      at Function.Module._load (module.js:276:25)
      at Function.Module.runMain (module.js:441:10)
      at startup (node.js:139:18)
      at node.js:968:3
  npm ERR! Linux 4.4.14-24.50.amzn1.x86_64
  npm ERR! argv "/opt/elasticbeanstalk/node-install/node-v4.4.6-linux-x64/bin/node" "/opt/elasticbeanstalk/node-install/node-v4.4.6-linux-x64/bin/npm" "--production" "install"
  npm ERR! node v4.4.6
  npm ERR! npm  v2.15.5
  npm ERR! code ELIFECYCLE

  npm ERR! knex@0.12.4 postinstall: `node ./scripts/build.js lib "npm run build"`
  npm ERR! Exit status 1
  npm ERR!
  npm ERR! Failed at the knex@0.12.4 postinstall script 'node ./scripts/build.js lib "npm run build"'.
  npm ERR! This is most likely a problem with the knex package,
  npm ERR! not with npm itself.
  npm ERR! Tell the author that this fails on your system:
  npm ERR!     node ./scripts/build.js lib "npm run build"
  npm ERR! You can get information on how to open an issue for this project with:
  npm ERR!     npm bugs knex
  npm ERR! Or if that isn't available, you can get their info via:
  npm ERR!
  npm ERR!     npm owner ls knex
  npm ERR! There is likely additional logging output above.

  npm ERR! Please include the following file with any support request:
  npm ERR!     /tmp/deployment/application/npm-debug.log
  Running npm install:  /opt/elasticbeanstalk/node-install/node-v4.4.6-linux-x64/bin/npm
  Setting npm config jobs to 1
  npm config jobs set to 1
  Running npm with --production flag
  Failed to run npm install. Snapshot logs for more details.
  Traceback (most recent call last):
    File "/opt/elasticbeanstalk/containerfiles/ebnode.py", line 695, in <module>
      main()
    File "/opt/elasticbeanstalk/containerfiles/ebnode.py", line 677, in main
      node_version_manager.run_npm_install(options.app_path)
    File "/opt/elasticbeanstalk/containerfiles/ebnode.py", line 136, in run_npm_install
      self.npm_install(bin_path, self.config_manager.get_container_config('app_staging_dir'))
    File "/opt/elasticbeanstalk/containerfiles/ebnode.py", line 180, in npm_install
      raise e
  subprocess.CalledProcessError: Command '['/opt/elasticbeanstalk/node-install/node-v4.4.6-linux-x64/bin/npm', '--production', 'install']' returned non-zero exit status 1 (ElasticBeanstalk::ExternalInvocationError)
caused by: + /opt/elasticbeanstalk/containerfiles/ebnode.py --action npm-install
```

**Comment by tgriesser** [comment]
Sorry about that, fixed in 0.12.5

**Comment by love8587** [comment]
@tgriesser Thanks a lot !!
